api-catalog-ui/frontend/src/components/ServiceTab/ServiceTab.test.jsx

```diff
@@ -31,10 +31,7 @@ const selectedService = {
     apiVersions: ['org.zowe v1', 'org.zowe v2'],
     defaultApiVersion: ['org.zowe v1'],
     ssoAllInstances: true,
-    gatewayUrls: {
-        'org.zowe v1': 'api/v1',
-        'org.zowe v2': 'api/v2',
-    },
+    apis: { 'org.zowe v1': { gatewayUrl: 'api/v1' } },
```

Comment:
Why only 'api/v1' is present? in apis?

Reply:
this is only a test, the way it works is no different if there is v1 or v2, it is taking a value from map based on key